api/src/pcapi/core/offers/offer_metadata.py

```diff
@@ -60,6 +62,13 @@ def _get_event_metadata_from_offer(offer: offers_models.Offer) -> Metadata:
         firstBeginningDatetime: datetime.datetime = offer.firstBeginningDatetime
         event_metadata["startDate"] = firstBeginningDatetime.isoformat(timespec="minutes")
 
+    event_metadata["eventAttendanceMode"] = (
+        "OnlineEventAttendanceMode" if offer.isDigital else "OfflineEventAttendanceMode"
+    )
+
+    if offer.extraData and offer.extraData.get("releaseDate"):
+        event_metadata["validFrom"] = str(offer.extraData["releaseDate"])
```

Comment:
(style) utiliser deux syntaxes différentes pour lire dans un dict est un peu bizarre, et le cast en str me parait superflu juste pour satisfaire mypy
```suggestion
    release_date = offer.extraData.get("releaseDate") if offer.extraData else None
    if release_date:
        event_metadata["validFrom"] = release_date
```